supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -294,6 +294,16 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*:
 
+[[session-persistence]]
+==== image:images/yes.png[yes] session persistence (noun)
+*Description*: _Session persistence_, also known as a sticky session, is a process in which a load balancer sends all requests in a user session to a specific network server. Session persistence can improve performance and network resource usage. Depending on which term your audience is most familiar with, use either "session persistence" or "sticky session" consistently.
```

Comment:
```suggestion
*Description*: _Session persistence_, also known as a _sticky session_, is a process in which a load balancer sends all requests in a user session to a specific network server. Session persistence can improve performance and network resource usage. Depending on which term your audience is most familiar with, use either "session persistence" or "sticky session" consistently.
```
I'm not sure if this is an issue, but do we need to italicize references to other glossary entries? Or put them in quotation marks? Or just leave them as-is? I couldn't find specific guidance for this situation.

Reply:
@bergerhoffer thoughts? I'm leaning towards not italicizing because this is part of the term definition. I use quotes later when discussing usage.

Reply:
I looked elsewhere in the SSG and other places where we say "also known as <x>" we do italicize <x> most times. It doesn't really matter if it's another glossary entry or not. 

One or two places we do use quotes, but I think italics are more appropriate in this situation

Reply:
@mportman12 - What are you thoughts on how best to resolve this issue in this PR?

Reply:
Added italics